CRM/Core/Smarty/plugins/function.crmKey.php

```diff
@@ -29,8 +29,5 @@
  *   the generated key
  */
 function smarty_function_crmKey($params, &$smarty) {
-  return CRM_Core_Key::get(
-      $params['name'],
-      CRM_Utils_Array::value('addSequence', $params, 0)
-    );
+  return CRM_Core_Key::get($params['name'], $params['addSequence'] ?? FALSE);
```

Comment:
This is better because `CRM_Core_Key::get()` expects a boolean not an int.

Reply:
agree